Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json

```
@@ -98,6 +98,10 @@
             "name": "omni-equipped-NECK",
             "directions": 4
         }
```

needs to be formatted correctly
```suggestion
        },
```